package.json

```diff
@@ -3,13 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/react-fontawesome": "^0.1.16",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "axios": "^0.23.0",
     "node-sass": "^6.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.3.0",
```

Comment:
it would be good to upgrade react router to v6. Either follow this guide https://reactrouter.com/docs/en/v6/upgrading/v5 
Or create new branch with and add v6 there.

Reply:
This might also help you https://www.youtube.com/watch?v=zEQiNFAwDGo